internal/netsync/manager.go

```diff
@@ -1275,6 +1343,19 @@ func (m *SyncManager) needTx(hash *chainhash.Hash) bool {
 	return true
 }
 
+// needMixMsg returns whether or not the mixing message needs to be downloaded.
+func (m *SyncManager) needMixMsg(hash *chainhash.Hash) bool {
+	if m.rejectedMixMsgs.Contains(hash[:]) {
+		return false
+	}
+
+	if m.cfg.MixPool.HaveMessage(hash) {
+		return false
+	}
+
```

Comment:
Just to clarify -- confirmed as in a finished run?  So that we don't re-add back, say, stale PRs from already completed runs?

Reply:
Yeah.  Obviously it wouldn't survive a server restart or go back very far, which is the same as what recently confirmed txns done.

It's just to help prevent additional traffic if malicious peers are trying to gum up the works.  Such messages would be rejected during validation, but the intention of the check in `needTx` and similar for the one here is to even avoid the need to request, receive, and the process them.

To be clear, I wasn't suggesting that is implemented now, rather just to add a comment that it would be ideal to implement in the future.